src/include/autoinstall/autopart.rb

```diff
@@ -281,14 +281,16 @@ def try_add_boot(conf, disk)
       )
       tc = Builtins.eval(conf)
       if !@planHasBoot && root &&
-          (Ops.greater_than(
-            Ops.get_integer(disk, "cyl_count", 0),
-            Partitions.BootCyl
-          ) ||
-            Arch.ia64 ||
-            Arch.ppc ||
-            Arch.sparc)
+         (Ops.greater_than(
+           Ops.get_integer(disk, "cyl_count", 0),
+           Partitions.BootCyl) ||
+           # If it is a ppc but not a baremetal Power8 system (powerNV).
+           # powerNV do not have prep partition and do not need any because
+           # they do not call grub2-install (bnc#989392).
+           (Arch.ppc && !Arch.board_powernv)
+         )
         pb = {}
+        # PPCs do not need /boot but prep partition only.
         if !Arch.ppc
```

Comment:
well, what happen if board is powernv and cryptic first condition is true? then it create prep partition. so maybe use quick exit here?
like 
```
# comment about powernv
return tc if Arch.ppc && Arch.board_powernv
if !@planHasBoot && root &&
  ....